.stickler.yml

```diff
@@ -0,0 +1,5 @@
+linters:
```

Comment:
This file should be the one from the linters-config repo: https://github.com/microverseinc/linters-config/blob/master/css/.stickler.yml